go.mod

```diff
@@ -6,6 +6,7 @@ toolchain go1.22.5
 
 require (
 	github.com/hashicorp/go-plugin v1.6.2-0.20240829161738-06afb6d7ae99
+	github.com/leanovate/gopter v0.2.10-0.20210127095200-9abe2343507a
```

Comment:
It looks like [v0.2.11](https://github.com/leanovate/gopter/releases/tag/v0.2.11) is tagged - would that be more stable?